### PR TITLE
Improved the confing file to use JSON

### PR DESCRIPTION
**Pull request by lukechronc**
Changed the way the configuration is stored according to #8 to use JSON.

### CODE REVIEW

**Comment by lukechronc** [comment]
This should fix the backwards compatibility

File: runtime-decompiler/src/main/java/org/jrd/backend/data/Config.java [hunk 1]
```diff
@@ -4,26 +4,33 @@
...
+import com.google.gson.Gson;
+
 
 /**
  * Singleton class for storing and retrieving configuration strings.
  */
 public class Config {
 
     private static Config config;
+    private static Gson gson;
```
Comment (judovana) [inline runtime-decompiler/src/main/java/org/jrd/backend/data/Config.java]: just brainstroming - any reason to have it  class wide ?
Likely no, but is slightly better then local wide:) Can you makeit final?

**Comment by judovana** [comment]
any deire to add tests?

File: runtime-decompiler/src/main/java/org/jrd/backend/data/Config.java (hunk 1 above)
Comment (lukechronc) [reply runtime-decompiler/src/main/java/org/jrd/backend/data/Config.java]: Yeah you're probably right

**Comment by lukechronc** [comment]
> any deire to add tests?

Add tests for checking the functionality of loadConfigFile() and saveConfigFile()?

**Pull request merged**